redaxo/src/addons/phpmailer/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+Version 2.5.0 – 24.08.2018
+--------------------------
+
+### Neu
+
+* E-Mail-Benachrichtigung bei Fehlern (@phoebusryan, @skerbis)
```

Comment:
Kannst es jetzt hier drin lassen, aber als Hinweis für die Zukunft: Besser Version und Changelog nicht anfassen. Dann ist es für mich leichter beim Releasen, einheitlich vorzugehen.
So schreibe ich das Datum zum Beispiel in der Beta erst als XX.XX.2018, und beim eigentlichen Release suche ich alle XX.XX.2018 und setzt das richtige Datum. Daher bitte zumindest das Datum in XX.XX.2018 ändern.

Reply:
Ok